src/executor.cpp

```diff
@@ -96,8 +96,7 @@ void Executor::Run(void* arg) {
 
   try {
     rcl_wait_set_t wait_set = rcl_get_zero_initialized_wait_set();
-    rcl_ret_t ret = rcl_wait_set_init(&wait_set, 0, 2, 0, 0, 0, 0,
-        executor->context_, rcl_get_default_allocator());
+    rcl_ret_t ret = rcl_wait_set_init(&wait_set, 0, 2, 0, 0, 0, rcl_get_default_allocator());
```

Comment:
It seems that the `rcl_wait_set_init` function has the parameters as follows https://github.com/ros2/rcl/blob/master/rcl/include/rcl/wait.h#L124